src/js/experiments/gettingStartedGuide.js

```diff
@@ -23,30 +23,72 @@ function showHomePageBadges() {
   }
 }
 
-function setUpTracking() {
-  const badges = Array.from($('.wrapper-link'));
-  badges.forEach((badge) => {
-    badge.addEventListener('click', () => {
-      window.AnalyticsClient.trackAction('clicked-landing-page-badge', {
-        badgeText: badge.innerText,
-        badgeHref: badge.href,
+function setUpTracking(variation) {
+  if (window.location.pathname === '/docs/2.0/getting-started/') {
+    window.AnalyticsClient.trackAction('visited-gettingStarted-page', {
+      variation,
+      referrer: document.referrer,
+    });
+  }
+
+  // Adding tracking to a tags that link to /docs/2.0/getting-started/
+  const linksToGettingStarted = Array.from(
+    $("li > a[href='/docs/2.0/getting-started/']"),
+  );
+  linksToGettingStarted?.forEach((link, i) => {
+    let linkLocation;
+    if (i === 0) linkLocation = 'mobile-sidebar';
+    if (i === 1) linkLocation = 'sidebar';
+    if (i === 2) linkLocation = 'homePage-section-link';
```

Comment:
This would be a great [switch/case](https://www.w3schools.com/js/js_switch.asp) statement ;)

Reply:
Ive ALWAYS wanted to use that, cant believe I missed a prime opportunity 😭. I can do a refactor!